src/librustc_target/Cargo.toml

```diff
@@ -11,6 +11,7 @@ path = "lib.rs"
 [dependencies]
 bitflags = "1.0"
 log = "0.4"
+cc = "1.0.1"
```

Comment:
Oh sorry I meant moreso that `cc` is laready linked elsewhere and it's already handled in probing for it, so we should reuse that location to add paths to link as opposed to doing so in the target spec directory here. (as this is how MSVC is handled in general)

Reply:
Sorry I'm still not clear on what to do. 

You want me to patch cc-rs or librustc_codegen_ssa or both?

Reply:
Sorry so what I mean is that `librustc_codegen_ssa` already (AFAIK) has tons of logic for dealing with MSVC and leverages the `cc` crate to find compilers and add paths to link against. The logic for handling UWP paths should all happen around there rather than being duplicated here.

Reply:
Code is not pretty... but it seems to work. Let me know what you think